patterns/disposer.md

```diff
@@ -0,0 +1,84 @@
+# The Promise Disposer Pattern
```

Comment:
Well, you've linked to issues that were opened by you, so it might look like something that you are promoting